packages/discord-verify/src/node.ts

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+// @ts-ignore-error Node Crypto types are not well defined yet
+import crypto from "node:crypto";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+// @ts-ignore-error Node Process
+import {
+	isValidRequest as verifyRequest,
+	PlatformAlgorithm,
+} from "./lib/verify";
+import type { Request, SubtleCryptoImportKeyAlgorithm } from "./types";
+export { hexStringToBinary, PlatformAlgorithm, verify } from "./lib/verify";
+
+/**
+ * Validates a request from Discord. If you are not on the latest version 16 or 18 of Node, you should pass a specific value in for the algorithm.
```

Comment:
This isn't necessarily true, I think it should stay similar to this, but mention that it'll try for you.

Reply:
I think I'm going to leave it in to dissuade people from trying, so there isn't a performance hit

Reply:
That's why I said leave it, but add. So like: It will try for you at the cost of performance